### PR TITLE
Bumping natsort to 5.0.2

### PR DESCRIPTION
**Pull request by holtgrewe**

### CODE REVIEW

**Comment by holtgrewe** [comment]
@synapticarbors It looks like the same thing is happening here.

**Comment by synapticarbors** [comment]
Something is still going on here. I think when PR #2 added a branch to the main feedstock repo, it generated actual packages:

https://anaconda.org/conda-forge/natsort/files

I'm not sure how to unwind this so I'm pinging @conda-forge/core for assistance.

**Comment by synapticarbors** [comment]
@jakirkham @ocefpaf Can either of you comment on what should be done here?

**Comment by jakirkham** [comment]
TL;DR A branch was added with the new version on the feedstock. That resulted in some packages being release pre-merge. This will happen any time one adds a branch directly to the main repo. To avoid this one should fork the feedstock, make changes in their fork, and PR them back to the main repo. Given the current state of things, my recommendation is to merge this PR and delete the extra branch.

> Something is still going on here.

This shares the same commit sha as the other PR. So we are seeing the old statuses here too.

> I think when PR #2 added a branch to the main feedstock repo, it generated actual packages

Yes, this is why we ask that people fork before making changes and PR changes from their forks. Though if anyone has an idea about how to document this better or make it clear, we are definitely open to suggestions.

> Can either of you comment on what should be done here?

What's the objective? At this point the packages are released with the exception of a few AppVeyor packages. Given that, my suggestion would be to merge this PR and delete the extra (non-`master`) branch. Sound reasonable?

**Comment by jakirkham** [comment]
This has been superseded by other PRs that have bumped the version passed this one.
